pvlib/pvsystem.py

```diff
@@ -3031,3 +3031,144 @@ def combine_loss_factors(index, *losses, fill_method='ffill'):
         combined_factor *= (1 - loss)
 
     return 1 - combined_factor
+
+
+def nonuniform_irradiance_deline_power_loss(
+    rmad,
+    model: Literal[
+        "fixed-tilt", "single-axis-tracking"
+    ] = "single-axis-tracking",
+    fillfactor_ratio: float = None,
+):
+    r"""
+    Estimate the power loss due to irradiance non-uniformity.
+
+    This model is described for bifacial modules in [1]_, where the backside
+    irradiance is less uniform due to mounting and site conditions.
+
+    Depending on the mounting type, the power loss is estimated with either
+    equation (11) or (12) of [1]_. Passing a custom polynomial is also valid.
+
+    Use ``fillfactor_ratio`` to account for different fill factors between the
```

Comment:
I think ``fillfactor_ratio`` would be easier for users if it was `fillfactor : fill factor at standard test condition (STC) of the module` and the scaling by 0.79 happened inside the function.

Reply:
I considered that option, but in that case it shouldn't be used when the user specifies their custom polynomial, since it may be created with another fillfactor value. Giving the opportunity to use it in that case is of my taste, thou I don't have an strong opinion and I will change it to raise an exception.